shared/common-adapters/standard-screen.desktop.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+import React from 'react'
+import {Box, Text, Icon} from '../common-adapters'
+import {globalStyles, globalColors, globalMargins} from '../styles/style-guide'
+import type {Props} from './standard-screen'
+
+const defaults = {
+  showClose: true,
+}
+
+const StandardScreen = (props: Props) => {
+  const {children, onClose, showClose, errorMessage} = {...defaults, ...props}
+  return (
+    <Box style={styleContainer}>
+      {!!onClose && showClose && <Icon style={styleClose} type='iconfont-close' onClick={() => onClose()} />}
```

Comment:
I was thinking that an API consumer may want to pass and onClose handler and forget about it, but then change if the button should show up or not from some boolean state (such as waiting for some non cancellable action). It felt weird requiring the consumer to provide and not-provide the onClose handler to signify that. But maybe it's not, I'm happy to remove it. I don't feel very strongly about this.